packages/apps/transfer/src/services/accounts/get-transactions.ts

```diff
@@ -0,0 +1,36 @@
+export interface Transaction {
+  fromAccount: string;
+  height: number;
+  amount: string;
+  crossChainId?: number;
+  toAccount: string;
+  blockTime: string;
+  requestKey: string;
+  token: string;
+  blockHash: string;
+  idx: number;
+  chain: number;
+  crossChainAccount?: string;
+}
+
+export async function getTransactions(options: {
+  network: string;
+  chain: string;
+  account: string;
+}): Promise<Transaction[]> {
+  const { network, chain, account } = options;
+
+  try {
+    const result: Transaction[] = await fetch(
+      `https://estats${
```

Comment:
It would be nice to have this whole string in a separate const

Reply:
Agreed. I checked constants/kadena.ts but it's heavily focussed on the chainweb API. Would be better maybe to rewrite this in another task to also be able to make it focussed on multiple APIs, like estats. I will create a separate task for this on the board.